src/components/Image/index.tsx

```diff
@@ -53,21 +48,20 @@ function Image(props) {
     );
 }
 
-function imagePropsAreEqual(prevProps, nextProps) {
+function imagePropsAreEqual(prevProps: ImageProps, nextProps: ImageProps) {
     return prevProps.source === nextProps.source;
 }
 
-Image.propTypes = imagePropTypes;
-Image.defaultProps = defaultProps;
+Image.resizeMode = RESIZE_MODES;
```

Comment:
`resizeMode` was set to `ImageWithOnyx`, not `Image`.

Reply:
Same as above, `ImageWithOnyx` doesn't have the `resizeMode` property

Reply:
Updated!

Reply:
Removed!